code/datums/noobmode.dm

```diff
@@ -0,0 +1,129 @@
+/datum/action/equipHazard
+	name = "Equip skinsuit"
+	desc = "hazard gear and internals. Takes a few seconds."
+	icon_icon = 'icons/obj/clothing/suits.dmi'
+	button_icon_state = "skinsuit"
+	var/obj/item/clothing/suit/space/skinsuit/suitslot
+	var/obj/item/clothing/mask/breath/maskslot
+	var/obj/item/clothing/head/helmet/space/helmetslot
+	var/obj/item/tank/internals/airslot
+	var/obj/item/storage/box/survBox
+	var/paniced = FALSE
+
+/datum/action/equipHazard/Trigger()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		testing(paniced)
+		if(!paniced)
+			if(pannickEquip(H))
+				toggle()
+		else
+			if(stowEquipment(H))
+				toggle()
+		//var/obj/item/held = H.get_active_held_item()
+		/*if(!held)
+			held = H.get_inactive_held_item()
+			if(held && istype(held,/obj/item/storage/box/survival))
+				to_chat(H,"<span class='warning'>You are putting on the outfit inside [held], hold still!</span>")
+				pannickEquip(H)
+			else
+				to_chat(H,"<span class='warning'> you must be holding the box!</span>")
+				return
+		else
+			if(held && istype(held,/obj/item/storage/box/survival))
+				to_chat(H,"<span class='warning'>You are putting on the outfit inside [held], hold still!</span>")
+				pannickEquip(H)
+				*/
+
+
+/datum/action/equipHazard/proc/GatherItems(obj/item/storage/CNT)
```

Comment:
could use a better var name here

Reply:
CNT meant for content, if you got another suggestion ill commit it

Reply:
I changed my mind actually, it's pretty close to cunt so that's a +1 from me

Reply:
I guese thats resolved